elide-contrib/elide-dynamic-config-helpers/src/main/java/com/yahoo/elide/contrib/dynamicconfighelpers/DynamicConfigHelpersUtil.java

```diff
@@ -0,0 +1,217 @@
+/*
+ * Copyright 2020, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.contrib.dynamicconfighelpers;
+
+import com.yahoo.elide.contrib.dynamicconfighelpers.model.ElideSecurityConfig;
+import com.yahoo.elide.contrib.dynamicconfighelpers.model.ElideTableConfig;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import org.everit.json.schema.Schema;
+import org.everit.json.schema.loader.SchemaLoader;
+import org.hjson.JsonValue;
+import org.json.JSONObject;
+import org.json.JSONTokener;
+
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.charset.Charset;
+import java.nio.file.DirectoryStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+@Slf4j
+/**
+ * Util class for Dynamic config helper module.
+ */
+public class DynamicConfigHelpersUtil {
+
+    public static final String SCHEMA_TYPE_TABLE = "table";
```

Comment:
Instead of making the SCHEMA_TYPE strings - make an enum that takes a constructor argument which is the POJO class you expect to deserialize.  Then you have a nice mapping between the enum name and the POJO class and you don't need so many switch statements below.

Reply:
In the same enum, you can tie together three bits of information - a friendly name, the POJO class to deserialize, and the JSON file where the schema lives.

Reply:
wont be needed due to refactoring of the class.